bip_metainfo/src/builder/mod.rs

```diff
@@ -161,6 +173,57 @@ impl<'a> MetainfoBuilder<'a> {
         self
     }
 
+    /// Get decoded value of announce-list key
+    pub fn get_trackers(self) -> Option<Vec<Vec<String>>> {
```

Comment:
Can you update these to take &self? It doesnt look like we are consuming any members of self.

Same for the other getters.

Reply:
done